README.rst

```diff
@@ -3,6 +3,9 @@ codespell
 
 Fix common misspellings in text files. It's designed primarily for checking
 misspelled words in source code, but it can be used with other files as well.
+It does not check for word membership in a complete dictionary, but instead
+looks for a set of common misspellings. Therefore it might catch errors like
+"adn", but it will not catch "adnasdfasdf".
```

Comment:
I still think it would be nice to make it a bit more positive, for example by saying "it also shouldn't generate false-positives when you use a niche term it doesn't know about".

Reply:
Improved suggestion.
```suggestion
"adn", but it will not catch "adnasdfasdf". This also means it shouldn't
generate false-positives when you use a niche term it doesn't know about.
```